clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/content/RatingContentView.kt

```diff
@@ -24,6 +26,11 @@ class RatingContentView(context: Context, renderer: TemplateRenderer, extras: Bu
         remoteView.setImageViewResource(R.id.star4, R.drawable.pt_star_outline)
         remoteView.setImageViewResource(R.id.star5, R.drawable.pt_star_outline)
 
+        // Request Codes for all stars are passed as an extra to cancel all pending intents when any of the star is clicked
+        extras.putIntArray(PTConstants.KEY_REQUEST_CODES, IntArray(5) {
```

Comment:
rather than passing in extras we can pass on the generated int in method signature

Reply:
@CTLalit Do you mean to the getPendingIntent() function? This disturbs the current structure of the code?

Reply:
We can create a function overload, or have a default value for the param which will not disturb existing code.

Reply:
@CTLalit This can be done but seems like an overkill for now in my opinion. I don't see where else will this new parameter could be needed later